HandsOnLab/README.md

```diff
@@ -85,11 +85,11 @@ private void OnPropertyChanged([CallerMemberName] string name = null) =>
 private void OnPropertyChanged([CallerMemberName] string name = null)
 {
     var changed = PropertyChanged;
+    
+    if (changed == null)
```

Comment:
Since C# 6 this can be a oneliner:
changed?.Invoke(this, new PropertyChangedEventArgs(name));